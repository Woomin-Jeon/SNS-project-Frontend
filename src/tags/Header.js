const Header = (n) => {
  return n;
};

export default Header;
