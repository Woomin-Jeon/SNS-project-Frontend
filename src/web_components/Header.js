import SearchBox from "../components/headers/SearchBox";
import HeaderHome from "../components/headers/HeaderHome";
import React from "react";

function Header({
  loginState,
  setLoginState,
  currentUserState,
  setCurrentUserState,
  setSearchState,
}) {
  return (
    <div className="header">
      <SearchBox
        loginState={loginState}
        setLoginState={setLoginState}
        setSearchState={setSearchState}
      />
      <HeaderHome
        loginState={loginState}
        setLoginState={setLoginState}
        currentUserState={currentUserState}
        setCurrentUserState={setCurrentUserState}
      />
    </div>
  );
}

export default Header;
