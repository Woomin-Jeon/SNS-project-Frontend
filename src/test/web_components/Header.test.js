import Header from "../../web_components/Header";
import React from "react";
import {shallow} from 'enzyme';

describe('Header', () => {
  let loginState;
  let setLoginState;
  let currentUserState;
  let setCurrentUserState;
  let setSearchState;

  it('renders well', () => {
    const component = shallow(
      <Header
        loginState={loginState}
        setLoginState={setLoginState}
        currentUserState={currentUserState}
        setCurrentUserState={setCurrentUserState}
        setSearchState={setSearchState}
      />
    );

    expect(component).toMatchSnapshot();
  });
});
