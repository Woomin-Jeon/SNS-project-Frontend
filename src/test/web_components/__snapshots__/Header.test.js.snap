// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders well 1`] = `ShallowWrapper {}`;
